contrib/kpatch.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description="Apply kpatch kernel patches"
+
+[Service]
+Type=oneshot
```

Comment:
Should it also have `RemainAfterExit=yes` so that it shows "active" and nobody tries to restart the service?